Orthtree/doc/Orthtree/Orthtree.txt

```diff
@@ -256,15 +284,78 @@ purposes.
 
 For nontrivial point counts, the naive approach's calculation time dwarfs that of either the %orthtree or kd-tree.
 
+\section Section_Orthtree_Migration Migrating Code Written Before Release 6.0
+
+The orthtree package changed to allow for custom data stored per node in the orthtree. To migrate existing code written before \cgal 6.0 `Orthtree_traits_point` can be used for a point-based orthtrees. The aliases `CGAL::Quadtree` and `CGAL::Octree` have been extended by a boolean template parameter to allow for non-cubic cells, which is the default. The data is passed via the traits class and no longer directly to the orthtree.
+
+Former code to declare and define an Octree with cubic cells was as follows:
+\code{.cpp}
+typedef CGAL::Point_set_3<Kernel::Point_3> Point_set;
+typedef CGAL::Octree<Kernel, Point_set, Point_map> Octree;
+...
+Octree octree(points, points.point_map());
+\endcode
+
+\cgal 6.0 code with identical behavior is now:
+\code{.cpp}
+typedef CGAL::Point_set_3<Kernel::Point_3> Point_set;
+typedef CGAL::Octree<Kernel, Point_set, Point_map, false> Octree;
+...
+Octree octree(points, points.point_map());
+\endcode
+
+The node class does not exist anymore and has been replaced by the lightweight type `Node_index`. All information formerly contained in the node class is now accessible via the `Orthtree` interface.
+
+Former node access was as follows:
```

Comment:
It would be helpful IMO to show how to access the points of a node in both versions

Reply:
@soesau did you see that comment? What do you think?